features/step_definitions/functional_tests/functional_tests_steps.rb

```diff
@@ -30,14 +30,18 @@
 Then(/^all the functional tests for the package "([^"]*)" pass$/) do |package_name|
   tests_dir = "/var/lib/#{package_name}-test"
   package_core_name = package_name.match(/python-(.+)/).captures.first
+  service_name = package_name.match(/python-(.*)(client)/).captures.first
   ssl_insecure =
     case package_name
     when "python-novaclient"
-      json_response = JSON.parse(admin_node.exec!("crowbar nova show default").output)
-      json_response["attributes"]["nova"]["ssl"]["insecure"]
+      json_response = JSON.parse(admin_node.exec!("crowbar #{service_name} show default").output)
```

Comment:
There is a helper for that, you can do: `proposal(service_name)`
The code is here: https://github.com/SUSE-Cloud/cct/blob/master/features/support/feature_helpers.rb#L4